tests/PHPUnit/Unit/ConfigTest.php

```diff
@@ -390,5 +390,47 @@ public function test_forceSave_writesNothingIfThereAreNoChanges()
 
         @unlink($configFile);
     }
+    
+    public function testFromGlobalConfig()
+    {
+        $userFile = PIWIK_INCLUDE_PATH . '/tests/resources/Config/config.ini.php';
+        $globalFile = PIWIK_INCLUDE_PATH . '/tests/resources/Config/global.ini.php';
+        $commonFile = PIWIK_INCLUDE_PATH . '/tests/resources/Config/common.config.ini.php';
+        
+        $config = Config::getInstance();
```

Comment:
The singleton instance shouldn't be used in unit tests; create a new instance instead, ie, `new Config(...)`. `setTestEnvironment` shouldn't be used for these tests as well.